### DISCUSSION TITLE
Install multiple helm charts with different values files in the same namespace?

### DISCUSSION THREAD

**Discussion by dev-samples** [fluxcd/flux2#1495] [Q&A] [answered]
Based on:

https://github.com/fluxcd/flux2-kustomize-helm-example/tree/main/apps/staging
https://github.com/fluxcd/flux2-kustomize-helm-example/blob/main/apps/base/podinfo/release.yaml

I have a namespace **podinfo** with one chart and app running = podinfo.

But what if I need to install multiple app/helm charts each with its separate values file in the same namespace?

I was not able to find an example doing that in:
https://github.com/fluxcd/flux2-kustomize-helm-example
https://fluxcd.io/docs/guides/helmreleases/

Am I missing something in the docs on how to do that?

**Comment by stevendick-work** [comment, marked as answer]
My project would need this support as well.

I haven't tried it yet, but my first attempt to handle this would be to define a **HelmRelease** per instance I want to install. This maps to how to currently handle this requirement with using Helm - the same chart deployed multiple times using a different release name and values.
